### PR TITLE
Added End-to-End Tests for supervision Comminity Event, Unpaid Work Requirement type & completed  UPW Sections - Individual's details & Diversity information

### PR DESCRIPTION
**Pull request by smalepati99**
Given I create new Offender in nDelius
And I create Supervision Community Event in Delius
And I add a requirement for this event with the type called "unpaid work"
And I create an entry in NOMIS (a corresponding person and booking in NOMIS)
And I start Unpaid Work Assessment from Delius
When I login to UPW and navigate to UPW Task List
And I click on "Individual's details" link
And I complete "Individual's details" Section
And I click on "Gender information" link
And I complete "Gender information" Section
And I click on "Cultural and Religious Adjustments" link
And I complete "Cultural and Religious Adjustments" Section

### CODE REVIEW

**Review by marcus-bcl — DISMISSED** [review]

File: steps/unpaidwork/login.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { expect, type Page } from '@playwright/test'
+
+export const login = async (page: Page) => {
+    // const [page] = await  Promise.all([page.waitForEvent('page')])
```
Comment (anthony-britton-moj) [inline steps/unpaidwork/login.ts]: Did you mean to leave this commented code? If no longer required, maybe delete?

File: steps/unpaidwork/login.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    const unpaidWorkTitle = 'Complete and download the Community payback assessment'
+    const title = await page.locator('title').textContent()
+
+    //may already be logged in
+    if (title.trim() == unpaidWorkTitle) {
+        page.once('dialog', dialog => dialog.accept())
+        await page.locator('a', { hasText: 'Sign out' }).click()
+    }
+
+    // await expect(page).toHaveTitle(/HMPPS Digital Services - Sign in/)
```
Comment (anthony-britton-moj) [inline steps/unpaidwork/login.ts]: And here?

File: steps/unpaidwork/login.ts (hunk 1 above)
Comment (smalepati99) [reply steps/unpaidwork/login.ts]: I forgot to delete the commented line. I will delete it

File: steps/unpaidwork/login.ts (hunk 2 above)
Comment (smalepati99) [reply steps/unpaidwork/login.ts]: I uncommented this one, thanks

**Review by anthony-britton-moj — APPROVED** [review]

File: steps/unpaidwork/login.ts (hunk 1 above)
Comment (smalepati99) [reply steps/unpaidwork/login.ts]: I have now deleted and pushed it

File: steps/unpaidwork/login.ts (hunk 2 above)
Comment (smalepati99) [reply steps/unpaidwork/login.ts]: I uncommented and pushed this one

**Pull request merged**